genesyscloud/tfexporter/tfstate_exporter.go

```diff
@@ -140,57 +146,6 @@ func (t *TFStateFileWriter) writeTfState() diag.Diagnostics {
 		log.Print(cliErrorPostscript)
 		// Don't fail everything even if this errors.
 
-		err := t.writeTfStateLegacy(stateFilePath)
-		if err != nil {
-			return nil
-		}
-		return nil
-	}
-	return nil
-}
-
-func (t *TFStateFileWriter) writeTfStateLegacy(stateFilePath string) error {
```

Comment:
All of this function is extraneous as it is handled in logging cliErrorPostscript above